float64 gain_p
float64 gain_i
float64 gain_d
float64 current_val
float64 target_val
float64 previous_error
float64 previous_integrator_val
float64 previous_derivative_val
float64 alpha
float64 integral_term_min
float64 integral_term_max
float64 dt
---
float64 current_integrator_val
float64 current_derivative_val
float64 current_error
float64 u
